gen/gen.go

```diff
@@ -0,0 +1,145 @@
+package main
+
+import (
+	"fmt"
+	"reflect"
+	"runtime"
+	"strings"
+
+	"github.com/dave/jennifer/jen"
+	"github.com/filecoin-project/specs-actors/actors/builtin/account"
+	"github.com/filecoin-project/specs-actors/actors/builtin/cron"
+	init_ "github.com/filecoin-project/specs-actors/actors/builtin/init"
+	"github.com/filecoin-project/specs-actors/actors/builtin/market"
+	"github.com/filecoin-project/specs-actors/actors/builtin/miner"
+	"github.com/filecoin-project/specs-actors/actors/builtin/multisig"
+	"github.com/filecoin-project/specs-actors/actors/builtin/paych"
+	"github.com/filecoin-project/specs-actors/actors/builtin/power"
+	"github.com/filecoin-project/specs-actors/actors/builtin/reward"
+)
+
+// This is a tool used to save some time writing all the message builder methods for actors.
+// It produces code that does NOT compile but saves a lot of typing.
+func main() {
```

Comment:
a tool used to generate the methods in `chain/account.go`, `chain/cron.go`, `chain/multisig.go`, etc.